src/components/custom-field-input-date/custom-field-date.css

```diff
@@ -1,12 +1,18 @@
 @import '../../styles/spacing.css';
 @import '../../styles/colors-v2.css';
+@import '../../styles/typography-v2.css';
 
 .date-input input::-webkit-inner-spin-button,
 .date-input input::-webkit-calendar-picker-indicator {
   display: none;
   -webkit-appearance: none;
 }
 
+.date-input input {
```

Comment:
This doesn't have bad implications in this use-case but FYI: we like to avoid using nested generic selectors -- we can just boil this down to using nested selectors.

The reason it is _not good_ to use generic selectors: it discourages composition. 

If we apply styling to all nested HTML `<input>`s in a component, if the component accepted a React node as a child and rendered it, we could be introducing styling pollution across components. 

If we apply styling to all nested HTML `<input>`s in a class name, if the class name is composed into another component, we could be introducing styling pollution to its own HTML markup. 

It is usually better to styling each individual element via CSS modules. We want to encourage composition, and the less gotchas the better.